packages/manager/modules/pci/src/projects/project/training/dashboard/dashboard.controller.js

```diff
@@ -0,0 +1,111 @@
+import flatten from 'lodash/flatten';
+import map from 'lodash/map';
+import filter from 'lodash/filter';
+import illustration from '../assets/partner.png';
+
+export default class PciTrainingDashboardController {
+  /* @ngInject */
+  constructor(CucCloudMessage, CucRegionService, atInternet) {
+    this.CucCloudMessage = CucCloudMessage;
+    this.CucRegionService = CucRegionService;
+    this.atInternet = atInternet;
+  }
+
+  $onInit() {
+    this.illustration = illustration;
+    // eslint-disable-next-line prefer-destructuring
+    this.currentRegion = this.regions[0];
```

Comment:
```[this.currentRegion] = this.regions;``` no need to disable eslint on this line

Reply:
yeah, but maybe we prefer to have a more readable syntax ?

Reply:
@XciD I personally found the syntax suggested by @frenautvh more elegant
Moreover I think we should avoid disabling eslint as much as possible, as there is an alternative way to do so without disabling it, we should go with it

Reply:
I totally agree with you about the elegancy of destructuring. However, the javascript way of accepting it for just one value is kind of not natural, you shouldn't be able to unpack an array of multiple element to only one value. For example, in python, if you do that and the array have multiple value, you will throw an error. So for only one value, I think it's more clean and readable.
But, it's your project. Will change 👍